kafka_utils/kafka_rolling_restart/main.py

```diff
@@ -367,6 +412,34 @@ def validate_opts(opts, brokers_num):
     return False
 
 
+def get_task_class(tasks, task_args):
+    """Reads in a list of tasks provided by the user,
+    loads the appropiate task, and returns two lists,
+    pre_stop_tasks and post_stop_tasks
+    :param tasks: list of strings locating tasks to load
+    :type tasks: list
+    :param task_args: list of strings to be used as args
+    :type task_args: list
+    """
+    pre_stop_task = []
+    post_stop_task = []
+    task_args_itr = task_args.__iter__()
+    tasks_classes = [PreStopTask, PostStopTask]
+
+    for func in tasks:
+        for task_class in tasks_classes:
+            imported_class = dynamic_import(func, task_class)
+            if imported_class:
+                if task_class is PreStopTask:
```

Comment:
We are importing classes using dynamic_import, and I havent formed instances at this point, hence I cannot use isinstance for this check.